examples/00-mapdl-examples/BodyFlux-Averaging.py

```diff
@@ -0,0 +1,324 @@
+"""
+.. _ref_solenoid_magnetostatic_2d:
+
+=======================================
+Analysis of a 2D Magnetostatic Solenoid
+=======================================
+
+This example shows how to gather and plot results with material discontinuities
+across elements (Power graphics style) versus the default full average results
+(Full graphics style).
+
+
+Description
+===========
+
+Mechanical APDL has two averaging methods depending the method used for
+presenting results:
+
+* **Full graphics** presents the entire selected model with node averaged results.
+  In the case of an node shared by two or more elements that have differing
+  materials, the stress field is continuous across the element material boundary
+  (the shared nodes).
+
+* **Power graphics** presents the entire selected model with averaged results
+  within elements of the same material, and discontinuous across material
+  boundaries.
+
+There are other differences between the two methods; this example concentrates
+on material boundaries.
+
+The native PyMAPDL post-processing is like Full graphics with respect to
+material boundaries.
+
+Axisymmetric solenoid magnetostatic example
+-------------------------------------------
+
+The geometry of the solenoid is given in Figure 1.
+
+.. figure:: ../../../_static/model_solenoid_2d.png
+    :align: center
+    :width: 600
+    :alt:  Solenoid geometry description
+    :figclass: align-center
+
+    **Figure 1: Solenoid geometry description.**
+
+Load and boundary condition
```

Comment:
Should it be conditions (plural)? Is load a condition? If not, it is fine as is!

Reply:
Hey @PipKat 
That is an interesting question.  To me a load is different than a boundary condition.  But someone who learned first on WB Mechanical may disagree as its documentation puts 'Loads' under 'Boundary Conditions' as just another type of BC.  

I think it is more correct to say "Loads and boundary conditions".

Reply:
@mikerife I think @germa89 would agree. Later, he has a "Loads and boundary conditions"--so he's treating them differently and answered my question indirectly.

Reply:
Whatever you decide guys :)